src/main/java/frc/robot/constants/ShooterConstants.java

```diff
@@ -3,8 +3,14 @@
 public class ShooterConstants {
 
     // motor Ids
-    public static final int BOTTOM_MOTOR_ID = 1;
-    public static final int TOP_MOTOR_ID = 2;
+    public static final int RIGHT_MOTOR_ID = 1;
+    public static final int LEFT_MOTOR_ID = 2;
+
+    // RPMs
+
+    public static final double LEFT_SHOOT_RPM = 6000;
+    public static final double RIGHT_SHOOT_RPM = 4000;
```

Comment:
Was our shoot redesigned to have spin, or is there another reason you're running the motors at different speeds?

Reply:
Yes, we have spin now. I used RPMs from Mechanical Advantage's shooter which is mechanically similar to ours, but we're going to end up fine tuning these values anyways.